File: set1/mario.c
#include <cs50.h>
#include <stdio.h>

void hash(int n);

int main(void)
{
    // declare height variable
    int h;
    // get height from user
    do
    {
        h = get_int("Enter the Height: ");
    }
    while (h < 1 || h > 8);
    // print obstacle 
    for (int j = 1; j <= h; j++)
    {
        // Print the spaces "8 - j" times
        for (int k = 0; k < h - j; k++)
        {
            printf(" ");
        }
        hash(j);
        printf("  ");
        hash(j);
        printf("\n");
    }
}

// Print hashes "n" times
void hash(int n)
{
    for (int i = 0; i < n; i++)
    {
        printf("#");
    }
}

File: set6/mario/mario.py
from cs50 import get_int

# check to get user input
while True:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break

# Loop to print output
for i in range(n):
    a = i + 1
    print(" " * (n - a) + "#" * (a) + "  " + "#" * (a))